robomimic/envs/env_robosuite.py

```diff
@@ -57,6 +57,7 @@ def __init__(
                 for saving to a dataset (to save space on RGB images for example).
         """
         self.postprocess_visual_obs = postprocess_visual_obs
+        self.use_depth_obs = use_depth_obs
```

Comment:
good to add a comment in the docstring for what `use_depth_obs` does. other than that LGTM